scripts/process_crashes.sh

```diff
@@ -30,8 +30,14 @@ function cleanup {
 trap cleanup EXIT
 
 if [[ $# -eq 0 ]]; then
-    echo "Usage: process_crashes.sh CRASHID [CRASHID ...]"
-    exit 1
+    if [ -t 0 ]; then
```

Comment:
I never knew about `-t`. What it does is test to see if the following fd is a terminal. fd `0` is stdin. So then this is asking "is stdin a terminal?" If it is, then the user executed it. If it's not, then it's on the other side of a pipe from some command.